jsonpickle/pickler.py

```diff
@@ -125,25 +149,38 @@ def encode(
         use_base85=use_base85,
         fail_safe=fail_safe,
     )
-    return backend.encode(
-        context.flatten(value, reset=reset), indent=indent, separators=separators
-    )
+
+    # Slightly brute-force method of removing null values
+    # Works by calling the module json, decoding the new json back to a python dict, looping through the new dict
+    # and returning a new dict without None values
+    finalResult = backend.encode(context.flatten(value, reset=reset), indent=indent, separators=separators)
+    if nullValues:
+        return finalResult
+    else:
+        previousResult = oldJson.loads(finalResult)
+        if type(previousResult) is not dict:
+            return previousResult
+        newDict = {}
+        for k, v in previousResult.items():
+            if v is not None:
+                newDict[k] = v
+        return oldJson.dumps(newDict)
 
 
 class Pickler(object):
     def __init__(
-        self,
-        unpicklable=True,
-        make_refs=True,
-        max_depth=None,
-        backend=None,
-        keys=False,
-        warn=False,
-        max_iter=None,
-        numeric_keys=False,
-        use_decimal=False,
-        use_base85=False,
-        fail_safe=None,
+            self,
+            unpicklable=True,
+            make_refs=True,
+            max_depth=None,
+            backend=None,
+            keys=False,
+            warn=False,
+            max_iter=None,
+            numeric_keys=False,
+            use_decimal=False,
+            use_base85=False,
+            fail_safe=None,
```

Comment:
Keep commits separate.  These whitespaces changes are bad.  I think we're configured to use `black` here.  Please never change superfluous whitespace.